agent/src/discovery_handler_manager/diagrams/dh_registration.mmd

```diff
@@ -0,0 +1,20 @@
+sequenceDiagram
+    Discovery Handler ->> Registration endpoint: Registers
+    Registration endpoint->> DiscoveryHandlerRegistry: register_endpoint()
+    DiscoveryHandlerRegistry ->> DiscoveryHandlerRegistry: Add endpoint to registered handlers list
+    DiscoveryHandlerRegistry ->> DiscoveryHandlerRequest: notify all DiscoveryHandlerRequest
+    alt Discovery Handler name is the same as in Request
+        DiscoveryHandlerRequest ->>+ Kubernetes API: Solve discovery properties
```

Comment:
This could use some elaboration. Is this the equivalent of "[For each Akri Config for the DH] Get discovery handler properties (filters) from Akri Configuration"?
```suggestion
        DiscoveryHandlerRequest ->>+ Kubernetes API: Look up discovery properties in Akri Configuration
```

Reply:
Here solving discovery properties is looking up for config maps and secrets that are in the `discovery_properties` field as required, this is handled in details by the `discovery_property_solver` module